### PR TITLE
340 separate ds auth from sign in and store auth token

### PR DESCRIPTION
**Pull request by AlexNi245**

### CODE REVIEW

File: packages/lib/src/session/signInWithEtheruem.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const siweMessage = new SiweMessage({
+        //Get domain from widget i guess
+        domain: 'www.dm3.chat',
+        address: account,
+        statement,
+        uri: 'origin',
+        nonce: nonce.toString(),
+        version: '1',
+        chainId: 1,
+        issuedAt: new Date(12).getTime().toString(),
```
Comment (AlexNi245) [inline packages/lib/src/session/signInWithEtheruem.ts:26]: How should we implement issuedAt ? If we are doing it properly it'll no longer possible to create the encryption key in an deterministic way

File: packages/lib/src/session/SignIn/signProfile.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    personalSign: PersonalSign,
+    account: string,
+    stringifiedProfile: string,
+): Promise<string> {
+    try {
+        const profileCreationMessage =
+            getProfileCreationMessage(stringifiedProfile);
+        return await personalSign(provider, account, profileCreationMessage);
+    } catch (e) {
+        console.log(e);
```
Comment (hai-ko) [inline packages/lib/src/session/SignIn/signProfile.ts:16]: remove `console.log`

File: packages/react/src/sign-in/Connectors.ts [hunk 3]
```diff
@@ -138,143 +140,63 @@ export async function signIn(
...
-                type: ConnectionType.ChangeConnectionState,
-                payload: singInRequest.connectionState,
-            });
-        }
-        dispatch({
-            type: ConnectionType.ChangeConnectionState,
-            payload: singInRequest.connectionState,
-        });
-    }
+    console.log('Auth state', state.auth);
```
Comment (hai-ko) [inline packages/react/src/sign-in/Connectors.ts:200]: remove `console.log`

File: packages/react/src/ui-shared/RequestContacts.ts [hunk 4]
```diff
@@ -37,20 +74,19 @@ export async function requestContacts(
         retrievedContacts = await Lib.account.getContacts(
             connection,
             userDb,
+            deliveryServiceToken,
             createEmptyConversationEntry,
         );
     }
+    console.log('GOT CONTRACT ', retrievedContacts);
```
Comment (hai-ko) [inline packages/react/src/ui-shared/RequestContacts.ts:81]: remove `console.log`

**Review by hai-ko — CHANGES_REQUESTED** [review]

**Review by hai-ko — APPROVED** [review]

**Pull request merged**